### PR TITLE
[ar] Localize setup/production-environment/tools/_index.md

### PR DESCRIPTION
**Pull request by essamgouda97**
Added translation for `setup/production-environment/tools/_index.md`

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
/retitle [ar] Localize setup/production-environment/tools/_index.md

File: content/ar/docs/setup/production-environment/tools/_index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+no_list: true
+---
+
+هناك العديد من الطرق والأدوات لإعداد مجموعة كوبرناتيز الخاصة بك في الإنتاج. على سبيل المثال:
+
+- [kubeadm](/docs/setup/production-environment/tools/kubeadm/)
+
+- [kops](https://kops.sigs.k8s.io/): أداة تسمح بتوفير العمليات الآلية لإعداد المجموعة. للحصول على البرامج التعليمية وأفضل الممارسات وخيارات التكوين ومعلومات حول التواصل مع المجتمع، يرجى التحقق من [موقعهم](https://kops.sigs.k8s.io/) للحصول على التفاصيل.
+
+- [kubespray](https://kubespray.io/): مجموعة من السكربتات الخاصة  [بانسيبل](https://docs.ansible.com/)، [الجرد](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/ansible.md#inventory)، أدوات التوفير، والمعرفة المجالية لمهام إدارة عناقيد كوبرناتيز العامة. يمكنك التواصل مع المجتمع على قناة سلاك [#kubespray](https://kubernetes.slack.com/messages/kubespray/).
```
Comment (adowair) [inline content/ar/docs/setup/production-environment/tools/_index.md:13]:
```suggestion
مزيج من مكونات إدارة البنية التحتية للعناقيد، بما في ذلك مجموعة الأوامر، و[قوائم المحتوى](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/ansible.md#inventory)، وأدوات التوفير، والمعرفة المجالية، الخاصة ب[أنسيبل](https://docs.ansible.com/). يمكنك التواصل مع مجتمع kubespray على .سلاك ([#kubespray](https://kubernetes.slack.com/messages/kubespray/))
```

**Review by adowair — COMMENTED** [review]
Thank you @essamgouda97, I've suggested one major change to reword the last paragraph. Otherwise some minor nits and updating the spelling of "kubernetes" to match the [latest standard](https://github.com/kubernetes/website/issues/45078).

File: content/ar/docs/setup/production-environment/tools/_index.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+no_list: true
+---
+
+هناك العديد من الطرق والأدوات لإعداد عناقيد الكوبيرنيتيس الخاصة بك في بيئات الإنتاج. على سبيل المثال:
+
+- [kubeadm](/docs/setup/production-environment/tools/kubeadm/)
+
+- [kops](https://kops.sigs.k8s.io/): أداة لإعداد العناقيد آلياً. للحصول على البرامج التعليمية وأفضل الممارسات وخيارات التكوين ومعلومات حول التواصل مع المجتمع، يرجى مراجعة [موقعهم](https://kops.sigs.k8s.io/).
+
+مزيج من مكونات إدارة البنية التحتية للعناقيد، بما في ذلك مجموعة الأوامر، و[قوائم المحتوى](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/ansible.md#inventory)، وأدوات التوفير، والمعرفة المجالية، الخاصة ب[أنسيبل](https://docs.ansible.com/). يمكنك التواصل مع مجتمع kubespray على .سلاك ([#kubespray](https://kubernetes.slack.com/messages/kubespray/))
```
Comment (adowair) [inline content/ar/docs/setup/production-environment/tools/_index.md:13]: Looks like a list sign is missing
```suggestion
- مزيج من مكونات إدارة البنية التحتية للعناقيد، بما في ذلك مجموعة الأوامر، و[قوائم المحتوى](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/ansible.md#inventory)، وأدوات التوفير، والمعرفة المجالية، الخاصة ب[أنسيبل](https://docs.ansible.com/). يمكنك التواصل مع مجتمع kubespray على .سلاك ([#kubespray](https://kubernetes.slack.com/messages/kubespray/))
```

**Review by adowair — CHANGES_REQUESTED** [review]
One minor nit, but otherwise looks good

**Pull request merged**